samples/caf_sensor_manager/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -36,6 +36,10 @@
 
 ipc0: &cpuapp_cpuppr_ipc {
```

Comment:
This looks a bit odd, isnt this the ipc between APP and PPR? Same in other overlays for 54H.

Reply:
@KAGA164 , can you provide some insight here?